docs/src/multi-physics_coupling.md

```diff
@@ -0,0 +1,37 @@
+# [Multi-physics coupling](@id multi-physics-coupling)
+A complex simulation can consist of different spatial domains in which
+different equations are being solved, different numerical methods being used
+or the grid structure is different.
```

Comment:
```suggestion
A complex simulation can consist of separate spatial domains in which, for
example, different equations are solved, different numerical methods are used,
the mesh structure is different, or some combination.
```

Reply:
Same as above: coupling is part of a different PR and has been removed from this one.